src/components/VRecentSearches/VRecentSearches.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    class="flex flex-col gap-1 rounded-sm border border-dark-charcoal-20 bg-white p-4 shadow-el-2"
+  >
+    <div class="flex flex-row items-center justify-between py-2">
+      <!-- Left margin to align with the text of recent searches. -->
+      <span class="category mx-2 my-1">
+        {{ $t('recent-searches.heading') }}
+      </span>
+      <VButton
+        v-show="entries.length"
+        variant="plain"
+        size="small"
+        class="caption-bold hover:underline"
+        @click="handleClear"
+      >
+        {{ $t('recent-searches.clear') }}
+      </VButton>
+    </div>
+
+    <ul
```

Comment:
I tried adding the Recent searches to the mobile searchbar in #1882, and I have problems with keyboard navigation.
Because the options are a `<li>` items, it's not possible to focus them using the Tab key. What is the reason for not using interactive `button` or `link` elements here?

Reply:
Because they should not be tabbable as per ARIA guidelines.

Reply:
This is also how the content switcher currently works (vertical arrow keys instead of tab).

Reply:
Thank you for clarifying! It turns out I forgot to add the `keydown` handler to the `input`, so pressing Arrow keys on the input wasn't working as supposed to. With the handler, it works really well!